csharp/src/Microsoft.ML.OnnxRuntime/NativeOnnxTensorMemory.cs

```diff
@@ -122,6 +130,8 @@ public NativeOnnxTensorMemory(IntPtr onnxValueHandle)
             }
         }
 
+        public IntPtr Handle { get { return _onnxValueHandle; } }
```

Comment:
Can this be marked as internal instead of public? Should not expose as public API

Reply:
sure

Reply:
I see -- was trying to minimize public API surface. If it is exposed as stable API,  it must be supported indefinitely.

Reply:
It is public - but in general, I can't see anyone using this class directly. It is a class mostly for other ORT classes to use (when it has to manage the lifecycle of native objects) and as far as supporting it indefinitely goes - it should be okay to do as long as this class maintains a handle to the native object (which it should as this is the main objective of this class). Open to suggestions though...

Reply:
It's a public property of an internal class, so this is not exposed to user.

I think the only change in this PR that affects the public interface is making the constructor of `DisposableNamedOnnxValue` from protected to private.

Reply:
@fs-eire -- good point. The class is marked as 'internal', so handle's visibility will be at most 'internal' (and not exposed publicly)